epics-service/src/test/java/edu/gemini/epics/ReadOnlyClientEpicsChannelTest.java

```diff
@@ -154,5 +155,22 @@ public void valueChanged(String channelName, List<Integer> values) {
 
     }
 
+    @Test
+    @Ignore
+    public void testSetReadRetries() throws CAException, TimeoutException {
```

Comment:
I added this test because I first thought I was not setting the number of retries correctly.